ci/jenkins/testkomodo.sh

```diff
@@ -12,5 +12,39 @@ install_package () {
 }
 
 start_tests () {
-    pytest -k "not test_typing"
+    pytest -k "not test_typing and not spe1"
+    run_spe1_tests
+}
+
+run_spe1_tests() {
+  pushd tests/data/spe1_st
+
+  echo "Initiating Ert run for Spe1 with new storage enabled..."
+  ert ensemble_experiment --enable-new-storage spe1.ert
+
+  echo "Ert run finished"
+
+  echo "Run the storage server api ..."
+  ert api &
+  ert_api_pid=$!
+  max_sec=60
+  until [ -f storage_server.json ]
+  do
+      if [ "$max_sec" -le 0 ]
+      then
+          echo "ert-storage has not started properly!"
+          exit 1
+      fi
+      sleep 1
+      max_sec=$(( max_sec-1 ))
+  done
+
+  echo "Test for accessing Spe1 data ..."
+  popd
+
+  pytest -k "spe1"
```

Comment:
In case of an error, this will exit without closing the `ert-storage` server. Maybe it's not an issue though ....

Reply:
we could align and at least do it the same way i think 👍  i can adjust to what you have !

Reply:
I injected this:
```sh
    set +e
    pytest -vs -k spe1
    status=$?
    echo "Close the storage server api ... "
    kill $ert_api_pid
    wait $ert_api_pid
    echo "Completed"
    if [ "$status" -g 0 ]; then
        exit $status
    fi
```